src/test/features/claimant-response/helpers/settlementHelper.ts

```diff
@@ -49,7 +49,7 @@ describe('settlementHelper', () => {
       expect(partyStatement.madeBy).to.be.eql('DEFENDANT')
       expect(partyStatement.type).to.be.eql('OFFER')
       expect(partyStatement.offer).is.not.undefined
-      expect(partyStatement.offer.content).to.be.eql('John Smith will pay the full amount, no later than 31 December 2050')
+      expect(partyStatement.offer.content).to.be.eql('John Smith will pay £200, no later than 31 December 2050')
```

Comment:
There is a test case .it('should return defendant party statement by installment option'. I see that the assertion for the offer content is missing. Can you add that as it related to these changes

Reply:
@srineelesh good point. Have fixed it.